os/net/bluetooth/bt_conn.c

```diff
@@ -718,6 +835,15 @@ void bt_conn_release(FAR struct bt_conn_s *conn)
 	bt_addr_le_copy(&conn->dst, BT_ADDR_LE_ANY);
 }
 
+struct bt_conn_s * bt_conn_relref(FAR struct bt_conn_s *conn)
```

Comment:
bt_conn_relref is the original function name? or bt_conn_release_ref is a better name for this function

Reply:
bt_conn_relref in not original name. I will modify it to bt_conn_release_ref .